src/overlays/actors/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -167,21 +167,149 @@ static u16 D_8082ABEC[] = {
+u8 gItemIconAgeInfo[] = {
+    AGE_REQ_CHILD, // Deku sticks
+    AGE_REQ_NONE,  // Deku nuts
+    AGE_REQ_NONE,  // Bombs
+    AGE_REQ_ADULT, // Bow
+    AGE_REQ_ADULT, // Fire arrows
+    AGE_REQ_NONE,  // Dins fire
+    AGE_REQ_CHILD, // Slingshot
+    AGE_REQ_NONE,  // Fairy Ocarina
+    AGE_REQ_NONE,  // Ocarina of Time
+    AGE_REQ_NONE,  // Bombchus
+    AGE_REQ_ADULT, // Hookshot
+    AGE_REQ_ADULT, // Longshot
+    AGE_REQ_ADULT, // Ice arrows
+    AGE_REQ_NONE,  // Farore's Wind
+    AGE_REQ_CHILD, // Boomerang
+    AGE_REQ_NONE,  // Lens of truth
+    AGE_REQ_CHILD, // Bean
+    AGE_REQ_ADULT, // Megaton Hammer
+    AGE_REQ_ADULT, // Light arrows
+    AGE_REQ_NONE,  // Nayrus love
+    AGE_REQ_NONE,  // Bottle
+    AGE_REQ_NONE,  // Red Potion
+    AGE_REQ_NONE,  // Green Potion
+    AGE_REQ_NONE,  // Blue Potion
+    AGE_REQ_NONE,  // Fairy
+    AGE_REQ_NONE,  // Fish
+    AGE_REQ_NONE,  // Milk
+    AGE_REQ_NONE,  // Letter in a bottle
+    AGE_REQ_NONE,  // Blue Potion
+    AGE_REQ_NONE,  // Bug
+    AGE_REQ_NONE,  // Big poe
+    AGE_REQ_NONE,  // Half Milk
+    AGE_REQ_NONE,  // Poe
+    AGE_REQ_CHILD, // Weird Egg
+    AGE_REQ_CHILD, // Chicken
+    AGE_REQ_CHILD, // Zelda's Letter
+    AGE_REQ_CHILD, // Keaton Mask
+    AGE_REQ_CHILD, // Skull Mask
+    AGE_REQ_CHILD, // Spooky Mask
+    AGE_REQ_CHILD, // Bunny Hood
+    AGE_REQ_CHILD, // Goron Mask
+    AGE_REQ_CHILD, // Zora Mask
+    AGE_REQ_CHILD, // Gerudo Mask
+    AGE_REQ_CHILD, // Mask of Truth
+    AGE_REQ_CHILD, // SOLD OUT
+    AGE_REQ_ADULT, // Pocket Egg
+    AGE_REQ_ADULT, // Pocket Cucco
+    AGE_REQ_ADULT, // Cojiro
+    AGE_REQ_ADULT, // Odd Mushroom
+    AGE_REQ_ADULT, // Odd Potion
+    AGE_REQ_ADULT, // Poacher's Saw
+    AGE_REQ_ADULT, // Goron's Sword (Broken)
+    AGE_REQ_ADULT, // Prescription
+    AGE_REQ_ADULT, // Eyeball Frog
+    AGE_REQ_ADULT, // Eye Drops
+    AGE_REQ_ADULT, // Claim Check
+    AGE_REQ_ADULT, // Unknown
+    AGE_REQ_ADULT, // Unknown
+    AGE_REQ_ADULT, // Unknown
+    AGE_REQ_CHILD, // Kokiri Sword
+    AGE_REQ_ADULT, // Master Sword
+    AGE_REQ_ADULT, // Big Goron's Sword
+    AGE_REQ_CHILD, // Deku Shild
+    AGE_REQ_NONE,  // Hylian Shield
+    AGE_REQ_ADULT, // Mirror Shield
+    AGE_REQ_NONE,  // Kokiri Tunic
+    AGE_REQ_ADULT, // Goron Tunic
+    AGE_REQ_ADULT, // Zora Tunic
+    AGE_REQ_NONE,  // Kokiri Boots
+    AGE_REQ_ADULT, // Iron Boots
+    AGE_REQ_ADULT, // Hover Boots
+    AGE_REQ_CHILD, // Bullet Bag 30
+    AGE_REQ_CHILD, // Bullet Bag 40
+    AGE_REQ_CHILD, // Bullet Bag 50
+    AGE_REQ_ADULT, // Quiver 30
+    AGE_REQ_ADULT, // Quiver 40
+    AGE_REQ_ADULT, // Quiver 50
+    AGE_REQ_NONE,  // Goron's Bracelet
+    AGE_REQ_NONE,  // Silver Gauntlets
+    AGE_REQ_NONE,  // Gold Gauntlets
+    AGE_REQ_CHILD, // Silver Scale
+    AGE_REQ_ADULT, // Gold Scale
+    AGE_REQ_ADULT, // Unused
+    AGE_REQ_NONE,  // Unused
+    AGE_REQ_NONE,  // Unused
+    AGE_REQ_ADULT  // Unused
```

Comment:
if this is indeed indexed by ItemID then you missed bomb bags

```suggestion
    AGE_REQ_ADULT, // Quiver 50
    AGE_REQ_NONE,  // Bomb Bag 20
    AGE_REQ_NONE,  // Bomb Bag 30
    AGE_REQ_NONE,  // Bomb Bag 40
    AGE_REQ_CHILD, // Goron's Bracelet
    AGE_REQ_ADULT, // Silver Gauntlets
    AGE_REQ_ADULT, // Gold Gauntlets
    AGE_REQ_NONE,  // Silver Scale
    AGE_REQ_NONE,  // Gold Scale
    AGE_REQ_ADULT  // Giant Knife / Biggoron's Sword
```

the values seem to make more sense this way too so it's probably right